app/uk/gov/hmrc/bindingtariffclassification/Module.scala

```diff
@@ -16,30 +16,37 @@
 
 package uk.gov.hmrc.bindingtariffclassification
 
+import javax.inject.{Inject, Provider}
 import play.api.inject.Binding
 import play.api.{Configuration, Environment}
 import uk.gov.hmrc.bindingtariffclassification.config.AppConfig
 import uk.gov.hmrc.bindingtariffclassification.crypto.LocalCrypto
 import uk.gov.hmrc.bindingtariffclassification.repository.{CaseMongoRepository, CaseRepository, EncryptedCaseMongoRepository}
-import uk.gov.hmrc.bindingtariffclassification.scheduler.{DaysElapsedJob, ScheduledJob, Scheduler}
+import uk.gov.hmrc.bindingtariffclassification.scheduler._
 import uk.gov.hmrc.crypto.CompositeSymmetricCrypto
 
 class Module extends play.api.inject.Module {
 
   override def bindings(environment: Environment, configuration: Configuration): Seq[Binding[_]] = {
     val appConfig = new AppConfig(configuration, environment)
-    val repositoryBinding: Binding[CaseRepository] = if(appConfig.mongoEncryption.enabled) {
+    val repositoryBinding: Binding[CaseRepository] = if (appConfig.mongoEncryption.enabled) {
       bind[CaseRepository].to[EncryptedCaseMongoRepository]
     } else {
       bind[CaseRepository].to[CaseMongoRepository]
     }
 
     Seq(
       bind[CompositeSymmetricCrypto].to(classOf[LocalCrypto]),
-      bind[ScheduledJob].to[DaysElapsedJob],
+      bind[ScheduledJobs].toProvider[ScheduledJobProvider],
       bind[Scheduler].toSelf.eagerly(),
       repositoryBinding
     )
   }
 
 }
+
+class ScheduledJobProvider @Inject()(activeDaysElapsedJob: ActiveDaysElapsedJob,
+                                     referredDaysElapsedJob: ReferredDaysElapsedJob)
+  extends Provider[ScheduledJobs] {
+  override def get(): ScheduledJobs = ScheduledJobs(Set(activeDaysElapsedJob, referredDaysElapsedJob))
```

Comment:
Didnt seem to work with just `Set[ScheduledJob]` so I had to wrap it in a case class `ScheduledJobs` :(